paddle/fluid/inference/tensorrt/engine.cc

```diff
@@ -99,6 +99,19 @@ void TensorRTEngine::DeclareOutput(const nvinfer1::ILayer* layer, int offset,
   buffer_sizes_[name] = 0;
 }
 
+void TensorRTEngine::DeclareOutput(const std::string& name) {
+  PADDLE_ENFORCE_EQ(0, buffer_sizes_.count(name), "duplicate output name %s",
+                    name);
+
+  auto* output = TensorRTEngine::GetITensor(name);
```

Comment:
Do all the outputs of a TensorRT layer can be retrieved by the name? Is this interface necessary given that there is already a `DeclareOutput`?

Reply:
> Do all the outputs of a TensorRT layer can be retrieved by the name? 

Yes, all the TensorRT layer will put its output into `itensor_map_`. The reason is that other TensorRT layer can get its input from `itensor_map_` directly.

For example:
https://github.com/luotao1/Paddle/blob/beb1245560b26fd198c3bdd7063334ad933f2d89/paddle/fluid/inference/tensorrt/convert/activation_op.cc#L32
```
 engine_->SetITensor(op.Output("Out")[0], layer->getOutput(0));
```
 > Is this interface necessary given that there is already a DeclareOutput?

Yes, we can easily use `engine->DeclareOutput("Out")` to DeclareOutput. Maybe this function can have another name.

Reply:
get